<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDairyWidget</class>
 <widget class="QWidget" name="CDairyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
     <item>
      <widget class="CDairyTagListView" name="listViewTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDairyDateTreeView" name="treeDairy">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="CDairyEditMainWidget" name="page_dairy"/>
     <widget class="CDairyStatisticsWidget" name="page_dairy_statistics"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDairyStatisticsWidget</class>
   <extends>QWidget</extends>
   <header>module_dairy/dairy_statistics/DairyStatisticsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CDairyEditMainWidget</class>
   <extends>QWidget</extends>
   <header>module_dairy/dairy_edit/DairyEditMainWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CDairyTagListView</class>
   <extends>QListView</extends>
   <header>module_dairy/dairy_tag/DairyTagListView.h</header>
  </customwidget>
  <customwidget>
   <class>CDairyDateTreeView</class>
   <extends>QTreeView</extends>
   <header>module_dairy/dairy_date/DairyDateTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
